### PR TITLE
Translate Damage HUD

### PR DESCRIPTION
**Pull request by Akuba-**
I have no clue if that works, I cannot test it right now. But I hope that will work.

### CODE REVIEW

**Comment by Deathreus** [comment]
Pretty sure putting the german translation underneath the english one is good enough, no need for multiple files

**Comment by 50Wliu** [comment]
@Deathreus 
> As of SourceMod 1.1, there is a new preferred method of shipping translations. By default, the main translation file should only contain English phrases. Additional translations are made in separate files, under a folder named after the ISO language code in languages.cfg.

https://wiki.alliedmods.net/Translations_(SourceMod_Scripting)

**Comment by 50Wliu** [comment]
That said I'm still debating whether to accept this for 1.10 or punt it until v2 instead.

File: addons/sourcemod/translations/freak_fortress_2.phrases.txt [hunk 1]
```diff
@@ -360,4 +360,14 @@
 		"#format"	"{1:i}"
 		"en"		"You have {1} detonations left!"
 	}
+	"Damage Other"
+	{
+		"#format"	"{1:N},{2:i}"
+		"en"		"{1}'s Damage: {2}"
+	}
+	"Damage Self"
```
Comment (winstliu) [inline addons/sourcemod/translations/freak_fortress_2.phrases.txt]: I would change this to "Your Damage Dealt" and the other one to "Spectator's Damage Dealt".

File: addons/sourcemod/translations/de/freak_fortress_2.phrases.txt [hunk 2]
```diff
@@ -345,4 +345,14 @@
 	{
 		"de"	"Du wurdest gegärtnert!"
 	}
+		"Damage Other"
```
Comment (winstliu) [inline addons/sourcemod/translations/de/freak_fortress_2.phrases.txt]: Extra indent here

**Comment by winstliu** [comment]
Closes #409.

File: addons/sourcemod/translations/freak_fortress_2.phrases.txt (hunk 1 above)
Comment (Akuba-) [reply addons/sourcemod/translations/freak_fortress_2.phrases.txt]: does ' work?

Comment (winstliu) [reply addons/sourcemod/translations/freak_fortress_2.phrases.txt]: Yes I believe it does

**Pull request merged**